pandas/io/formats/css.py

```diff
@@ -75,101 +130,79 @@ def __call__(self, declarations_str, inherited=None):
         props = dict(self.atomize(self.parse(declarations_str)))
         if inherited is None:
             inherited = {}
+        assert inherited is not None
```

Comment:
you really need this assert?

Reply:
No, indeed, that one I forgot. Thank you.